keyboards/ph_arisu/readme.md

```diff
@@ -0,0 +1,16 @@
+# PH ARISU
+
+![PH_ARISU](https://i.imgur.com/qr2Tn2X.jpg)
+
+PCB is base on https://github.com/FateNozomi/arisu-pcb
+and used for a local groupbuy in the Philippines.
+
+* Keyboard Maintainer: [Neil Brian Ramirez](https://github.com/NightlyBoards)
+* Hardware Supported: PH ARISU PCB
```

Comment:
This uses a Pro Micro, right?

Reply:
atmega32u4 on Pro Micro yes.

Reply:
Updated Readme https://github.com/qmk/qmk_firmware/pull/9612/commits/bb47f65e8b563d189bfd864137468ccba7a0fa1e